minimumLovablePrototype/helpers.py

```diff
@@ -73,7 +73,7 @@ def timedelta_2_seconds(time_delta: pd.Timedelta):
     integer_seconds = np.float64(round(time_delta.total_seconds()))
     fractional_seconds = np.float64(
         time_delta.delta - integer_seconds * constants.cNanoSecondsPerSecond
-    )
+    )/constants.cNanoSecondsPerSecond
```

Comment:
This appears to be fixed on `main`: https://github.com/jtec/prx/blob/d1e40a2d54c8b76de02a85ba8899dafe81d2cf9f/src/prx/helpers.py#L101